src/main/java/com/github/onsdigital/babbage/api/error/ErrorHandler.java

```diff
@@ -45,7 +48,12 @@ public static void handle(HttpServletRequest req, HttpServletResponse response,
         } else if (t instanceof LegacyPDFException) {
             error().exception(t).log("LegacyPDFException error");
             renderErrorPage(501, response);
-        } else {
+        }else if(t instanceof UnauthorizedException){
+            
+            error().exception(t).log("user is logged out");
+            renderErrorPage(401, response);
+        } 
+        else {
```

Comment:
```suggestion
        } else if (t instanceof UnauthorizedException) {
            error().exception(t).log("user is logged out");
            renderErrorPage(401, response);
        } else {
```
formatting is important for clarity and consistency.